package.json

```diff
@@ -3,10 +3,11 @@
 	"version": "0.0.1",
 	"license": "Apache-2.0",
 	"scripts": {
+		"preinstall": "npx only-allow pnpm",
```

Comment:
oh what does this do?

Reply:
It gives an error message in the terminal if you try to run yarn instead - handy as I always need the reminder for the first week or so!
<img width="384" alt="Screenshot 2024-06-19 at 10 51 23" src="https://github.com/guardian/commercial-templates/assets/108270776/90ab060c-e037-4466-8ebd-285f369c28e0">